specs/order_spec.rb

```diff
@@ -82,29 +82,49 @@
 xdescribe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      Grocery::Order.all.length.must_equal 100
     end
 
     it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
+      id = 1
+      food = { "Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9 }
+      order = Grocery::Order.new(id, food)
+      order.id.must_equal id
+      expected_count = 3
+      order.products.count.must_equal expected_count
+      order.products.must_include "Grape Seed Oil"
```

Comment:
This test is almost right. The big issue is that you don't call `Grocery::Order.all`, which means the test isn't exercising the code you're interested in. If you were to replace line 91 with

```ruby
order = Grocery::Order.all.first
```

it would be exactly what we're looking for. Remember, the _Act_ step of _Arrange_, _Act_, _Assert_ is always to call the method you're testing.